holidays/no.yaml

```diff
@@ -0,0 +1,17 @@
+---
+
+_nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=61.0&lon=8.0&zoom=8&addressdetails=1&accept-language=no,en'
+
+PH:  # https://en.wikipedia.org/wiki/Public_holidays_in_Norway
+  'Nyttårsdag': [1, 1]
+  'Skjærtorsdag': [easter, -3]
+  'Langfredag': [easter, -2]
+  'Påskedag': [easter, 0]
+  '2. Påskedag': [easter, 1]
+  '1. Mai': [1, 5]
+  'Grunnlovsdagen': [17, 5]
```

Comment:
First time I've tried this on Github so not sure where to actually make changes now.
I've only created or commented on issues in here before :)